crates/holochain_sqlite/src/sql/wasm/schema/0.sql

```diff
@@ -8,8 +8,8 @@ CREATE TABLE IF NOT EXISTS Wasm (
 );
 
 CREATE TABLE IF NOT EXISTS DnaDef (
-    hash            BLOB           PRIMARY KEY ON CONFLICT IGNORE,
-    blob            BLOB           NOT NULL
+    cell_id            BLOB           PRIMARY KEY ON CONFLICT IGNORE,
+    dna_def            BLOB           NOT NULL
 );
```

Comment:
Should not modify existing schema definitions. Add a new migration instead.

If this was a production ready tool instead of something we'd written ourselves, the hashes of the migrations would be placed in a table managed by the migration tool and the application would fail to start if any of the provided migrations changed hash.

Reply:
I know there are other breaking changes anyway, but I'd really like to be in the habit of making changes in a way that permits Holochain to upgrade successfully - even if there are other API incompatibilities.

Reply:
Though in this case it's a drop of the table and replacing it. There's no way to actually retain data here...

Reply:
It's not super complicated if you look at other schemas, but also not super required in this case IMO.

Reply:
> I'm just not familiar with how migrations

Look at the documentation for a real tool like flyway or liquibase or something like that. If you want to get a sense of how this works in general.

For this case, it really is just that when we make database changes, we provide the change as a new SQL file. That way, whatever state people started in, when they install a new conductor, they run all the new migrations and everyone ends up on the same exact schema eventually.

If you edit one, and I've already run it on my system and don't run it again, then people can end up in different database schema states.

Reply:
Okay, got it, thanks. I added a migration now: https://github.com/holochain/holochain/pull/5189/commits/98b7890ff6311ae945ff4068b7b6d6f586803574